README.md

```diff
@@ -1,156 +1,132 @@
+---
 
-## SFX:
+# Brought to you by
 
-Pipe98x (drawing cross)
+## Current contributors
 
-sirrealist2 (drawing circle/line)
+_Bottling up those happy thoughts!_
 
-## Testers & Platinum Troubleshooters:
+- [Orcaramelo](https://github.com/Orcaramelo): Project Lead, Lead Artist, Writer
+- [Nathan](https://github.com/TRIDENT1313): Lead Developer, Writer
+- [Sayori](https://www.youtube.com/channel/UCwH1ro8D7kE73jhkWU3APkA): Lead Writer, Musician
+- [Emilia/Monika (EYLO)](https://www.reddit.com/u/EmiliaMonika?utm_medium=android_app&utm_source=share): Writer
 
-AlexanDDOS
+## Previous contributors
 
-Leon9023 (Reddit, VK)
+_Thank you for your help in bringing FaE to life - we wouldn't be here without you!_
 
-Evgeny Zotov
+- [Otter](https://github.com/my-otter-self): Writer
+- [Nemu-sus](https://github.com/Nemu-sus): Artist
+- [NekoLaiS](https://github.com/NekoLaiS): Programmer
+- Sobbcat: Writer
+- SlightlyAmiss: Musician
+- [AlexanDDOS](https://github.com/AlexanDDOS): Project Lead, Lead Developer, Writer, Tester
+- ElReyZero: Programmer
+- Venus: Writer
+- tberriman: Proofreader
+- vermicellibug: Proofreader
+- Atlas: Proofreader
+- Lucian Pavel: Artist
+- Vova Petelin: Translator (Russian)
+- ElReyZero: Translator (Spanish)
+- littlebin: Translator (Chinese), Tester
+- Leon9023: Tester
+- Evgeny Zotov: Tester
+- MarioXenobladefan614: Tester
 
-littlebin
+## Special thanks
 
-MarioXenobladefan614
+- [Monika After Story Team](https://github.com/Monika-After-Story/MonikaModDev)
+  - Adapted source code and systems used with permission
+  - Weather assets used with permission
+  - Ongoing support
+- [Just Natsuki Team](https://github.com/Just-Natsuki-Team/NatsukiModDev)
+  - Adapted source code and systems used with permission
+  - Weather assets used with permission
+  - Ongoing support
+- [multimokia](https://github.com/multimokia): Night theme (_Slepyori_)
+- [/r/DDLC](reddit.com/r/ddlc/)
+- Pipe98x: SFX (drawing cross)
+- sirrealist2: SFX (drawing circle/line)
+- Fj0rd_: Prerelease Sayori spritework
+- **Our community, for your ongoing patience and support**
 
-</details>
+_You're all awesome!_
 
-<details>
-  <summary>License & Copyright</summary>
-  
-See "LICENSE" for license information
+---
 
-Made in accordance with Team Salvato's IP Guidelines
+# Community
 
-Copyright (c) 2018-2022 Alexander Osipov and the other creators
+Follow us on [Twitter!](https://twitter.com/JustSayoriDev)
 
-</details>
+Interested in taking part in a growing community of Sayori fans just like you, or just a chat with the team?
 
-### Community
-You can [follow us on twitter](https://twitter.com/JustSayoriDev) for game updates.
+If so, then please consider joining us on Discord: we'd love to see you there!
 
-Feel free to join our discord server too:
+**Please note we do not handle technical support or mod suggestions on Discord.**
 
- [![Discord](banner)](https://discord.gg/ZqfEddFKux)
+<br>
+<p align="center">
+    <a href="https://discord.gg/GswMG6RU">
```

Comment:
I bodged this together from the server ID and the existing link text. I noted in the commit message this would need to be updated by someone with the permissions to generate links: apologies that this wasn't made as clear as it should have been.

Can you provide a link?

Reply:
Sure thing
https://discord.gg/nUWzgVZ9WT